.github/workflows/test.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Download yt-dlp
         run: curl -L https://github.com/yt-dlp/yt-dlp-nightly-builds/releases/latest/download/yt-dlp -o yt-dlp && chmod +x ./yt-dlp
       - name: Install and build server dependencies
-        run: cd server/ && yarn install --frozen-lockfile && npx tsc
```

Comment:
All instances of yarn install --frozen-lockfile should be replaced wiith `npm ci` instead of `npm install`